includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -65,6 +65,19 @@
 				)
 			);
 
+			woocommerce_wp_text_input( array(
+				'id'                => '_low_stock_amount',
+				'value'             => '' === $product_object->get_low_stock_amount( 'edit' ) ? get_option( 'woocommerce_notify_low_stock_amount', 2 ) : $product_object->get_low_stock_amount( 'edit' ),
```

Comment:
Let's not set the global here. Doing so would mean you can never just have a global setting and not do it at product level.

Just use `$product_object->get_low_stock_amount( 'edit' )`. if it's blank we can use the global.

What you can add is a `placeholder` for the global. That way, when the product doesn't have this set, it will show whatever the global uses dynamically.